daemon/daemon.go

```diff
@@ -507,9 +507,6 @@ func (d *Daemon) Stop(sigCh chan<- os.Signal) error {
 		logger.Noticef("error writing maintenance file: %v", err)
 	}
 
-	d.snapdListener.Close()
-	d.standbyOpinions.Stop()
-
 	if d.snapListener != nil {
```

Comment:
should we move this too for consistency?

Reply:
I've taken the other suggestion instead of moving this, so we can use the waiting time to shutdown the snap listener